### ISSUE TITLE
Batch Normalisation

### ISSUE THREAD

**Issue by kannan500** [jiecaoyu/XNOR-Net-PyTorch#96]
https://github.com/jiecaoyu/XNOR-Net-PyTorch/blob/ebccb503b2f9ebc96187a75b6daa63d95b61d110/ImageNet/datasets/transforms.py#L58
In this line,you are describing Normalisation ,but I have a doubt...
@jiecaoyu How you find mean and variance during inference?

**Comment by kannan500** [comment]
In inference,Are you using the mean and variance from training ?

**Comment by jiecaoyu** [comment]
The mean and variance are computed using the script described [here](https://caffe.berkeleyvision.org/gathered/examples/imagenet.html). They will not change through the training.
